R/inference.R

```diff
@@ -16,6 +16,15 @@
 #' @param data_format A length 1 character vector, `"long"`, `"wide"` and
 #'   `"grouped"`. The format that the passed data is in. Defaults to `"long"`.
 #'   See `vignette("data-formats)` for details.
+#' @param long_data_names_spec A 3 element named list that specifies the names
+#'   of the three required columns in the long data format. The list must have
+#'   the required names:
+#'     * item: the name of the column containing the item indexes,
+#'     * rater: the name of the column containing the rater indexes,
+#'     * rating: the name of the column containing the ratings.
+#'   By default, the names of the columns are the same as the names of the
+#'   vector: `"item"`, `"rater"`, and `"rating"` respectively. This argument is
+#'   ignored when the `data_format` argument is either `"long"` or `"wide"`.
```

Comment:
"long" should be "grouped"

(I suggest writing them in the following order: "...is either wide or grouped")